BackEndFlask/models/queries.py

```diff
@@ -157,10 +171,18 @@ def get_users_by_team_id(team):
         ).all()
 
     except SQLAlchemyError as e:
-        error = e.__dict__['orig']
-        return error
+        logger.error(str(e.__dict__['orig']))
+        raise e
 
 def get_users_not_in_team_id(team):
+    """
```

Comment:
this has the same description that `get_users_by_team_id()` has.

Reply:
@malloc-nbytes I have updated the descriptions!